shared/workspace_repo/workspace.go

```diff
@@ -616,14 +641,14 @@ func (s *TFEWorkspace) GetEndpoints(ctx context.Context) (map[string]string, err
 	endpoints := map[string]string{}
 	outputs, err := s.GetOutputs(ctx)
 	if err != nil {
-		return nil, errors.Wrap(err, "Unable to get workspace outputs")
+		return nil, errors.Wrap(err, "unable to get workspace outputs")
 	}
 	if endpoint, ok := outputs["frontend_url"]; ok {
 		endpoints["FRONTEND"] = endpoint
-	} else if svc_endpoints, ok := outputs["service_endpoints"]; ok {
+	} else if svc_endpoints, ok := outputs["service_urls"]; ok {
```

Comment:
Is this going to be a problem with all the existing repos?

Reply:
No, the only place where service_endspoints was used was our examples lol

Reply:
Everywhere else has service_urls